pages/agent/clusters.md

```diff
@@ -53,6 +53,19 @@ To create additional queues:
 1. Enter a key and description.
 1. Select _Create Queue_.
 
+### Pause a queue
+
+Pausing a queue will prevent jobs from being dispatched to agents associated with that queue. This may affect builds targeting the paused queue.
```

Comment:
```suggestion
You can pause a queue to prevent jobs from being dispatched to agents associated with that queue. This may affect builds targeting the paused queue.
```

Just tossing around the idea of phrasing this in "You can" language like a few of the other sentences on this page, given this is an action we expect our users to take directly.

Reply:
I also wonder if we can make this sentence clearer?

> This may affect builds targeting the paused queue.

Does this mean trigger jobs from other pipelines targeting the queue to be paused? If so, perhaps we could include that specific detail too?

Reply:
@timriley yeah, I wasn't sure how much detail to go into here! With trigger jobs, they don't run on agents at all so don't really care about queues being paused. If the job they're triggering relies on the paused queue, it won't run while the queue is paused. If the job they're triggering targets another queue that is not paused, then it'll go ahead and run, unless an explicit dependency has been set

Reply:
I've tried a best pass at explaining in more detail. Let me know what you think :)